stetho/src/main/java/com/facebook/stetho/inspector/network/DefaultAsyncPrettyPrinterRegistry.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.network;
+
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.GuardedBy;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DefaultAsyncPrettyPrinterRegistry implements AsyncPrettyPrinterRegistry {
```

Comment:
Thanks, I will change the registry to non static and remove the interface as well